addons/takistan/CfgVehicles_TNA.hpp

```diff
@@ -0,0 +1,352 @@
+    linkedItems[] = {DEFAULT_ITEMS_RADIO, "CUP_V_PMC_CIRAS_Coyote_Grenadier", "CUP_H_OpsCore_Tan", "CUP_NVG_PVS15_black"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS_RADIO, "CUP_V_PMC_CIRAS_Coyote_Grenadier", "CUP_H_OpsCore_Tan", "CUP_NVG_PVS15_black"};
+    weapons[] = {"TACU_Takistan_W_M4A1_GL"};
+    respawnWeapons[] = {"TACU_Takistan_W_M4A1_GL"};
+    magazines[] = {mag_8("CUP_1Rnd_HEDP_M203"), mag_8("CUP_30Rnd_556x45_Stanag")};
+    respawnMagazines[] = {mag_8("CUP_1Rnd_HEDP_M203"), mag_8("CUP_30Rnd_556x45_Stanag")};
+};
+
+class TACU_Takistan_U_TNA_Commandos_Autorifleman: TACU_Takistan_U_TNA_Commandos_NCO {
+    displayName = "Autorifleman";
+    editorPreview = QPATHTOF(ui\Takistan_U_TNA_Commandos_Autorifleman.jpg);
+    backpack = "TACU_Takistan_B_TNA_LMG";
+    weapons[] = {"CUP_lmg_m249_para", "Throw", "Put"};
+    respawnWeapons[] = {"CUP_lmg_m249_para", "Throw", "Put"};
+    magazines[] = {"CUP_200Rnd_TE4_Red_Tracer_556x45_M249", mag_3("HandGrenade"), mag_2("SmokeShell")};
+    respawnMagazines[] = {"CUP_200Rnd_TE4_Red_Tracer_556x45_M249", mag_3("HandGrenade"), mag_2("SmokeShell")};
+};
+    
+ class TACU_Takistan_U_TNA_Commandos_Medic: TACU_Takistan_U_TNA_Commandos_NCO {
+    displayName = "Medic";
+    editorPreview = QPATHTOF(ui\Takistan_U_TNA_Commandos_Medic.jpg);
+    backpack = "tacs_Backpack_Kitbag_Medic_Sage";
+    attendant = 1;
+    linkedItems[] = {DEFAULT_ITEMS_RADIO, "CUP_V_B_Ciras_Olive4", "CUP_H_OpsCore_Spray", "CUP_NVG_PVS15_black"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS_RADIO, "CUP_V_B_Ciras_Olive4", "CUP_H_OpsCore_Spray", "CUP_NVG_PVS15_black"};
+    weapons[] = {"TACU_Takistan_W_M4A3", "CUP_hgun_M9", "Throw", "Put"};
+    respawnWeapons[] = {"TACU_Takistan_W_M4A3", "CUP_hgun_M9", "Throw", "Put"};
+    magazines[] = {mag_8("CUP_30Rnd_556x45_Stanag"), mag_3("CUP_15Rnd_9x19_M9"), mag_3("HandGrenade"), mag_2("SmokeShell")};
+    respawnMagazines[] = {mag_8("CUP_30Rnd_556x45_Stanag"), mag_3("CUP_15Rnd_9x19_M9"), mag_3("HandGrenade"), mag_2("SmokeShell")};
+};
+
+class TACU_Takistan_U_TNA_Commandos_Rifleman: TACU_Takistan_U_TNA_Commandos_NCO {
+    displayName = "Rifleman";
+    editorPreview = QPATHTOF(ui\Takistan_U_TNA_Commandos_Rifleman.jpg);
+    backpack = "CUP_B_Kombat_Olive";
+    linkedItems[] = {DEFAULT_ITEMS_RADIO, "CUP_V_B_Ciras_Olive", "CUP_H_OpsCore_Spray", "CUP_NVG_PVS15_black"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS_RADIO, "CUP_V_B_Ciras_Olive", "CUP_H_OpsCore_Spray", "CUP_NVG_PVS15_black"};
+    weapons[] = {"TACU_Takistan_W_M4A1", "CUP_hgun_M9", "CUP_launch_M136_Loaded", "Throw", "Put"};
+    respawnWeapons[] = {"TACU_Takistan_W_M4A1", "CUP_hgun_M9", "CUP_launch_M136_Loaded", "Throw", "Put"};
+    magazines[] = {mag_8("CUP_30Rnd_556x45_Stanag"), mag_3("CUP_15Rnd_9x19_M9"), mag_3("HandGrenade"), mag_2("SmokeShell"), "CUP_M136_M"};
+    respawnMagazines[] = {mag_8("CUP_30Rnd_556x45_Stanag"), mag_3("CUP_15Rnd_9x19_M9"), mag_3("HandGrenade"), mag_2("SmokeShell"), "CUP_M136_M"};
```

Comment:
Doesn't using `CUP_launch_M136` as the launcher work just fine, and then you wouldn't need to add a magazine?

Reply:
Magazine is always required to pre-load launchers for some reason. I did test this again to be 100% sure but without the rocket in the magazine list it's unloaded by default.

Reply:
That's the unloaded variant (`magazines[] = {"CBA_FakeLauncherMagazine"};`), as is the norm for newish CBA Disposable Launchers. Loaded one is `scope = 1` and should not be used directly, so this is indeed the correct way to do this.